test/unit/rules/no-negated-condition-test.js

```diff
@@ -221,6 +266,31 @@ generateRuleTests({
         `);
       },
     },
+    {
+      template:
+        '{{#unless (not (not condition))}}<img>{{else if (not (not condition))}}<input>{{/unless}}',
+      fixedTemplate:
+        '{{#unless condition}}<img>{{else if (not (not condition))}}<input>{{/unless}}',
```

Comment:
Should we keep this rule principle to not touch `{{else if` blocks, or try to fix double negation also in this case?

Reply:
I think we should touch `else if` too. Let's fix the double negation here.